docs/data/charts/zoom-and-pan/zoom-and-pan.md

```diff
@@ -143,14 +147,26 @@ The `zoomInteractionConfig` prop allows you to specify which interactions are en
 ```jsx
 <BarChartPro
   zoomInteractionConfig={{
-    // Enable both wheel and pinch zoom
-    zoom: ['wheel', 'pinch'],
+    // Enable wheel, pinch, and tap-and-drag zoom
+    zoom: ['wheel', 'pinch', 'tapAndDrag'],
     // Enable drag panning
     pan: ['drag'],
   }}
 />
 ```
 
+**Zoom** interactions:
+
+- **`wheel`**: Zoom in/out by scrolling the mouse wheel (default)
+- **`pinch`**: Zoom in/out by pinching on touch devices (default)
+- **`tapAndDrag`**: Zoom in/out by tapping once and then dragging vertically. Dragging up zooms in, dragging down zooms out.
+
+**Pan** interactions:
+
+- **`drag`**: Pan the chart by dragging with the mouse or touch (default)
+
+{{"demo": "ZoomAndPanInteractions.js"}}
```

Comment:
I was trying this demo on my phone and it seems I can only zoom when I tap and drag horizontally. When I do that, then I can drag vertically and it zooms in/out. If I  tap and drag vertically immediately, the page scrolls instead. I'll add a video, but it's probably a bit hard to see.

https://github.com/user-attachments/assets/6e0c0356-b4f1-4e6c-bcda-e50367be0c4a

Also, it seems a lot of the page is being selected, is that something we can fix? Not sure if a `preventDefault` would work, but worst case scenario we can apply a `user-select: none` to the body when dragging.

Reply:
Notice the same, I need to drag on horizontal before being able to zoom with vertical.

Surprisingly on desktop it's working fine.

Reply:
I've added behaviour to change touchAction after the tap is detected. Let me know if that fixes the issues. 

The selection doesn't seem to happen on my device, please check if it is still happening, as it shouldn't because the Surface has `userSelect: 'none'` by default.